pom.xml

```diff
@@ -109,4 +108,13 @@
 		<connection>scm:git:https://github.com/wso2/maven-tooling-bps.git</connection>
 		<tag>HEAD</tag>
 	</scm>
+	<properties>
+	<maven.plugin.plugin.version>3.4</maven.plugin.plugin.version>
```

Comment:
Fixed in https://github.com/wso2/maven-tooling-bps/pull/7/commits/43199d0a0b82d1b130de3da85e294ab25d422fd2